_posts/2017-09-18-episode-7-free-speech-limited.md

```diff
@@ -15,13 +15,15 @@ show_notes: |
+    **News clip:** A horrifying scene in Charlottesville as this car plowed into a crowd of people. The driver, then, backing up and witnesses say, "Dragging at least one person." [00:00:30] Unite the Right organizers-
+
+    **Veronica Belmont:** In August, a protest turned deadly in the American town of Charlottesville, Virginia. After organizing online, a large group of torch-wielding white supremacists descended on the town to protest a removal of a Confederate statue. They had assembled lawfully. They had every right to be there, and to express their opinion, and so did a large group of counter-protesters. Things got ugly, and in the end a woman named Heather Heyer was dead. Hit by [00:01:00] a car. Driven by one of the racist protestors. Laws that govern free expression differ depending on what country you live in. In the United States, freedom of speech is considered a fundamental virtue of democracy. There's a reluctance to restrict a person's right to say what's on their mind, and yet there's also a belief that free speech should not infringe on the freedom of others. After Charlottesville, this tension manifested [00:01:30] itself online, and with the Neo-Nazis website, the Daily Stormer, in particular. The site was accused of further inciting or encouraging violence. It was all too much for a group of tech companies, including GoDaddy, Google, and Cloudflare. They pulled their services, and, just like that, the Daily Stormer was offline, exiled to the dark web. Some will say this is an act of common decency, but it also brings up an important question: when large companies own the publication [00:02:00] platforms and the plumbing that makes the internet accessible to all of us, should they have the power to control our right to speak? That's what today's episode is exploring. I have three guests joining me for this conversation. There's Jillian York. She's with the Electronic Frontier Foundation and runs the website, onlinecensorship.org. She's in Berlin, Germany. Hi, Jillian.
 
     **Jillian York:** Hi!
 
-    **Veronica Belmont:** Also, here in the studio with me in San Francisco is Brandi Collins. She's a Senior Campaign Director with Color of Change, an online racial justice organization. Hi, Brandi.
+    **Veronica Belmont:** Also, here in the studio with me in San Francisco is Brandi Collins. She's a Senior Campaign Director with Color of Change, [00:02:30] an online racial justice organization. Hi, Brandi.
```

Comment:
I'm not sure all these inline timestamps should be here. Previous episodes don't seem to have them. It might mean a lot of manual cleanup to delete them all.

Reply:
I was told in Slack it was the final version, and then asked about the timestamps and pinged 3 people, none of whom responded.

I'm sure it would be better to remove them though. I'll do it.

Reply:
Michaela is removing them - will update PR shortly.